packages/api/test/__fixtures__/sdk/petstore/index.ts

```diff
@@ -163,36 +180,48 @@ class SDK {
    *
    * @summary Finds Pets by status
    */
-  get(path: '/pet/findByStatus', metadata: FindPetsByStatusMetadataParam): Promise<FindPetsByStatus_Response_200>;
```

Comment:
You'll notice that I've renamed these response types from `FinePetsByStatus_Response_200` to `FindPetsByStatusResponse200` as types should generally be pascal case.